ttnn/cpp/ttnn/operations/conv/conv2d/device/kernels/reader_conv_activations_2d_mcast_padded_with_halo_3x3_weights_v2.cpp

```diff
@@ -22,6 +22,26 @@ inline void print_pages(uint32_t l1_addr, uint32_t pagelen, uint32_t npages, uin
 }
 #endif
 
+template<int window_height, int window_width>
+FORCE_INLINE void read_dilated_channels(uint32_t& l1_write_addr_act, const uint32_t act_l1_read_addr, const uint32_t reader_channel_idx,
+        const uint32_t conv_act_c_bytes, const uint32_t stride_h_bytes, const uint32_t stride_w_bytes) {
+
+    uint32_t act_l1_read_addr_plus_offset = act_l1_read_addr + (reader_channel_idx * conv_act_c_bytes);
+    #pragma GCC unroll 3
```

Comment:
why unroll the outer loop and not the inner loop?

Reply:
Also why not use a constexpr unroll factor like elsewhere?

Reply:
Done.